__tests__/app.test.js

```diff
@@ -34,4 +34,32 @@ describe("GET /api", () => {
                 expect(response.body.endpoints).toHaveProperty("GET /api/topics"); 
             });
     });
-});
\ No newline at end of file
+});
+
+/*
+similarily to ticket 2, but modified for the specifics of the question,
+we could do something along the lines of first of, expecting the length of the object to be 8 or more,
+as we're working with 8 parameters, then we could just do 8 different expect toHaveProperty's for each one we was asked to include.
+That overall should cover everything we're looking for?
+*/
+
+describe("GET article by id", ()=>{
+    test("",()=>{
```

Comment:
This test block should have a description eg: "Status 200 responds with single article by id" This means that other developers will have a much better time understanding what the test suite is meant to prove